pq-crypto/s2n_pq.c

```diff
@@ -95,13 +95,21 @@ bool s2n_sikep434r2_asm_is_enabled() {
     return sikep434r2_asm_enabled;
 }
 
+bool s2n_pq_is_enabled() {
```

Comment:
Naming Nit: Flip the order of `pq` and `is`, so that it becomes `s2n_is_pq_enabled()`.

Reply:
I went back and forth on this. I was keeping this consistent with the previous style: https://github.com/awslabs/s2n/blob/cb734f6d3c4cde5ce18fbb48a8cb5d831a0f874f/pq-crypto/s2n_pq.h#L22

I think this way makes more sense when read as part of a conditional: `if (s2n_pq_is_enabled())` -> "If post-quantum is enabled".